homeassistant/components/tado/device_tracker.py

```diff
@@ -107,9 +107,9 @@ def update_state(self) -> None:
         _LOGGER.debug(
             "Updating Tado mobile device: %s (ID: %s)",
             self._device_name,
-            self._device_id,
+            self._attr_unique_id,
```

Comment:
I'd keep using a separate attribute for the device id. It has different semantics than the unique_id. The unique_id is used for the Home Assistant entity interface. The device id is used for the tado API interface.